RealtimeJMSToStream/README.rst

```diff
@@ -0,0 +1,85 @@
+Realtime Java Message Service (JMS) Source To Stream Sink Adapter Configuration
+===============================================================================
+
+The ETL Realtime Template can be used to create an Adapter that reads from a Realtime Source and pass it to a Sink to be processed further.
+In this example, we will read messages from Jave Message Service (JMS) source in realtime and use a Stream Sink to write the messages to CDAP Stream.
```

Comment:
It's better if the lines are not so long, but broken so they don't wrap. It makes commenting much easier, because then the comment can get closer to the relevant text.
```
messages from a Java Message Service (JMS)...the messages to a CDAP Stream.
```